helm/nessie/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
What is this for? Can it just go in the `README.md`?

Reply:
The purpose of this is file is to show something after deploying the Helm chart (and in this case it shows how to do port forwarding). `NOTES.txt` is going through the templating engine and will show a copy/pastable string to do the port forwarding, thus it can't go into a `README.md`